migrations/2023-07-04-093802_add_clp_currency_to_db/up.sql

```diff
@@ -0,0 +1,2 @@
+-- Your SQL goes here
+ALTER TYPE "Currency" ADD VALUE IF NOT EXISTS 'CLP' AFTER 'CHF';
```

Comment:
Convert the `Currency` enum to text in the database only. Create an issue an link it here to keep track of it.

Reply:
Raised #1705